test/js/node/dns/node-dns.test.js

```diff
@@ -230,7 +231,7 @@ test("dns.lookup (localhost)", done => {
   });
 });
 
-test("dns.getServers", done => {
+test.skipIf(isWindows)("dns.getServers", done => {
   function parseResolvConf() {
```

Comment:
☝️ We should put a TODO: comment here to get back to it later.

Side note: We should have a `test.todoIf()` like we have a `test.skipIf()`.